fish-rust/src/global_safety.rs

```diff
@@ -16,3 +16,9 @@ impl RelaxedAtomicBool {
         self.0.swap(value, Ordering::Relaxed)
     }
 }
+
+impl Clone for RelaxedAtomicBool {
```

Comment:
https://doc.rust-lang.org/std/sync/atomic/struct.AtomicBool.html does not implement Clone, so would probably be more appropriate to `impl From<bool>`, so you end up with `x.load().into()` vs `x.clone()`

Reply:
It's an interesting idea but the advantage of Clone is that it allows `#[derive(Clone)]` for other types containing `RelaxedAtomicBool`, which is what we're using here.

Presumably `AtomicBool` does not implement Clone because it is ambiguous which memory ordering to use; but RelaxedAtomicBool is specifically Relaxed so I think Clone is sensible.